schedule/schedule-appengine/src/main/client/jasify/admin/activity/admin-subscribe.html

```diff
@@ -96,14 +98,15 @@
     <div class="bootgrid-footer container-fluid">
         <div class="row">
             <div class="col-sm-6">
-                <pagination max-size="vm.maxSize" ng-model="vm.page" ng-change="vm.pageChanged()"
-                            total-items="vm.total" num-pages="vm.numPages" items-per-page="vm.perPage()"
+                <pagination max-size="vm.pagination.maxSize" ng-model="vm.pagination.page" ng-change="vm.pageChanged()"
+                            total-items="vm.pagination.total" num-pages="vm.pagination.numPages"
+                            items-per-page="vm.perPage()"
                             boundary-links="true" rotate="true">
                 </pagination>
             </div>
             <div class="col-sm-6 infoBar">
-                <div class="infos" translate="SHOWING_X_TO_Y_OF_Z_ENTRIES"
-                     translate-values="{fromEntry: '{{vm.perPage() * (vm.page - 1) + 1}}', toEntry: '{{[vm.perPage() * vm.page, vm.total] | min}}', totalEntries: '{{vm.total}}'}">
+                <pagination-info total-size="vm.pagination.total" current-page-size="vm.users.length"
+                                 items-per-page="vm.perPage()" page="vm.pagination.page"></pagination-info>
```

Comment:
Cool, I didn't have time to understand how `<pagination-info` worked when I did histories, so whenever you feel inspired you could give #653 a try :-)